scipy/stats/tests/test_stats.py

```diff
@@ -3111,6 +3113,13 @@ def test_precision_loss_gh15554(self):
             a[:, 0] = 1.01
             stats.skew(a)[0]
 
+    def test_empty_1d(self):
```

Comment:
Nd arrays are already handled by `_axis_nan_policy_decorator` and thoroughly tested in `test_axis_nan_policy::test_empty`. Only need to test 1d here.